docs/security/secret-detection.md

```diff
@@ -1,109 +1,7 @@
 # Secret Detection
 
-It is important to ensure that secrets don't make it in to the code base and compromise the security of our application data. The best way of doing so is by introducing a simple and easy way to detect secrets during its development, and gate for secrets in your CI/CD pipelines.
-This section describes how we can use the open-source secrets detection framework [Yelp Detect-Secrets](https://github.com/Yelp/detect-secrets), to accomplish this together with two tools from the Detect-Secrets framework created by the CSE Engineering Fundamentals and Security group.
-The Yelp Detect-Secrets tool utilizes a Shannon Entropy model to detect secrets in code bases and prevents new secrets from being entered.
+!!! warning
```

Comment:
Can we just remove this content and redirect all links (in the repo pointing to this) to point to the new content if applicable instead?

Reply:
Done! I was not sure if we wanted to keep the page around just to avoid broken links from outside the repo (like bookmarks). There were no other links to that page in the repo otherwise.

Reply:
@TessFerrandez is the deprecation ok with you, or should we delete this page completely?